ssl/t1_enc.c

```diff
@@ -175,6 +175,20 @@ int tls_provider_set_tls_params(SSL *s, EVP_CIPHER_CTX *ctx,
     return 1;
 }
 
+
+int EVP_CIPHER_iv_length_within_key_block(const EVP_CIPHER *c)
+{
+    int k;
```

Comment:
You can remove "k", and just replace the assignments with return statements, then remove the "else"

Reply:
Thank you, I have followed your advice.